Library/Homebrew/cmd/cleanup.rb

```diff
@@ -111,7 +111,11 @@ def cleanup_lockfiles
   end
 
   def rm_DS_Store
-    quiet_system "find", HOMEBREW_PREFIX.to_s, "-name", ".DS_Store", "-delete"
+    %w[Cellar Frameworks Library bin etc include lib opt sbin share var].each do |path|
+      path = HOMEBREW_PREFIX/path
+      next unless path.exist?
+      quiet_system "find", path.to_s, "-name", ".DS_Store", "-delete"
```

Comment:
You could give all paths to `find` to speed up the process, i.e. use only `quiet_system` call instead of one per directory.